Makefile

```diff
@@ -11,7 +11,7 @@ composer:
 dependencies:
 	make -s composer update -- --prefer-dist
 test:
-	$(RUNNER) "phpunit --coverage-text --coverage-html ./coverage"
+	$(RUNNER) "XDEBUG_MODE=coverage phpunit --coverage-text --coverage-html ./coverage"
```

Comment:
This fixes an issue where the env variable is missing, so the coverage isn't being printed out when running `make test`

<details>
<summary>Error: missing env</summary>

<img width="551" alt="Screenshot 2023-04-12 at 2 02 35 PM" src="https://user-images.githubusercontent.com/8046636/231419307-fc5d4793-3b04-4949-a63b-2feb2661e827.png">

</details>